src/angular/package.json

```diff
@@ -63,6 +71,7 @@
     "karma-jasmine": "~1.1.1",
     "karma-jasmine-html-reporter": "^0.2.2",
     "lint-staged": "^7.2.2",
+    "ng-packagr": "^4.2.0",
```

Comment:
Removal needed here.

Reply:
it wont build without it, angular-cli is looking for it, but i cant find it as a dependency anywhere...